phoenix-core/src/main/java/org/apache/phoenix/execute/MutationState.java

```diff
@@ -1198,6 +1199,7 @@ private void sendBatch(Map<TableRef, MultiRowMutationState> commitBatch, long[]
         Map<TableInfo, List<Mutation>> physicalTableMutationMap = Maps.newLinkedHashMap();
 
         // add tracing for this operation
+        long startTime = EnvironmentEdgeManager.currentTimeMillis();
```

Comment:
nitpick : the above L 1201 "add tracing" comment was for line L 1203 - maybe we can add your change just above that comment - to make the comment and code nearby as last time.